core/server/errors/no-database-found.js

```diff
@@ -0,0 +1,11 @@
+function NoDatabaseFound(message) {
+    this.message = message;
+    this.stack = new Error().stack;
+    this.statusCode = 404;
+    this.errorType = this.name;
+}
+
+NoDatabaseFound.prototype = Object.create(Error.prototype);
+NoDatabaseFound.prototype.name = 'NoDatabaseFound';
+
+module.exports = NoDatabaseFound;
```

Comment:
Hmm how about re-use `NotFoundError`?
I would like to avoid just saying "new Error(message)". because if start with logging, each error will get a level. ("normal", "critical" etc).